packages/dataviews/src/dataviews-layouts/list/index.tsx

```diff
@@ -319,6 +317,35 @@ function ListItem< Item >( {
 	);
 }
 
+function ViewListWrapper( {
+	children,
+}: {
+	children: ( store?: Ariakit.CompositeStore ) => React.ReactNode;
```

Comment:
If we really want to conceal the store, we might need to come up with some APIs that allow us to work with the store state directly. That might help simplify use cases like this, and allow us to refrain from going so low-level.